Cargo.toml

```diff
@@ -9,7 +9,6 @@ members = [
     "crates/cxx-qt-build",
     "crates/cxx-qt-gen",
```

Comment:
still need to make cxx-qt-gen only depend on cxx-qt headers / types and move them into cxx-qt.